src/app/loaders/express/index.ts

```diff
@@ -1,13 +1,16 @@
 import compression from 'compression'
+import connectDatadog from 'connect-datadog'
 import express, { Express } from 'express'
 import addRequestId from 'express-request-id'
+import { StatsD } from 'hot-shots'
 import http from 'http'
 import { Connection } from 'mongoose'
 import nocache from 'nocache'
 import path from 'path'
 import url from 'url'
 
 import config from '../../config/config'
+// import { datadogConfig } from '../../config/features/datadog.config'
```

Comment:
Is this commented import needed?

Reply:
Nope, that's an accident! Good catch! :)

Reply:
Pushed commit to fix! Thanks!